server/plugin/webhook.go

```diff
@@ -1045,3 +1049,40 @@ func (p *Plugin) handlePullRequestReviewNotification(event *github.PullRequestRe
 	p.CreateBotDMPost(authorUserID, message, "custom_git_review")
 	p.sendRefreshEvent(authorUserID)
 }
+
+func (p *Plugin) postStarEvent(event *github.StarEvent) {
+	repo := event.GetRepo()
+
+	subs := p.GetSubscribedChannelsForRepository(repo)
+
+	if len(subs) == 0 {
+		return
+	}
+
+	newStarMessage, err := renderTemplate("newRepoStar", event)
+	if err != nil {
+		p.API.LogWarn("Failed to render template", "error", err.Error())
+		return
+	}
+
+	post := &model.Post{
+		UserId:  p.BotUserID,
+		Type:    "custom_git_star",
```

Comment:
same for this, would it make sense to store all the texts like this in a const.

Reply:
LGTM